src/workerd/jsg/jsg.c++

```diff
@@ -499,4 +500,8 @@ bool isInGcDestructor() {
   return HeapTracer::isInCppgcDestructor();
 }
 
+bool USVString::isValidUtf8() const {
+  return simdutf::validate_utf8(cStr(), size());
```

Comment:
@anonrig I'm calling this from USVString's constructor in debug mode, so no one can accidentally pass off an invalid string as a USVString in the future.